<launch>
	<arg name="dev" default="/dev/input/js0"/>
	
	<!--node pkg="nodelet" type="nodelet" name="auv_base_nodelet_manager" args="manager"/-->
		
	<!--include file="$(find zeabus_teleop)/launch/includes/velocity_smoother.launch.xml"/-->
	<node pkg="joy" type="joy_node" name="joystick">
		<param name="dev" value="$(arg dev)" />
		<param name="deadzone" value="0.05" />		
	</node>
	
	<node pkg="zeabus_teleop" type="zeabus_joy" name="zeabus_joy">
	<!--<node pkg="zeabus_teleop" type="zeabus_joy" name="zeabus_joy">-->
		<param name="deadman_button" value="4" />
		<param name="roll_axis" value="0"/>
		<param name="roll_pitch" value="1"/>
		<param name="roll_yaw" value="3"/>
		<param name="depth_axis" value="4"/>
		<param name="topic_joy" value="/zeabus/cmd_vel"/>
	</node>
	
</launch>
